main.go

```diff
@@ -47,7 +47,7 @@ func resultWorker(g *libgobuster.Gobuster, filename string, wg *sync.WaitGroup)
 	var f *os.File
 	var err error
 	if filename != "" {
-		f, err = os.Create(filename)
+        f, err = os.OpenFile(filename, os.O_CREATE|os.O_RDWR|os.O_APPEND, 0660);
```

Comment:
We have a slight tabbing/formatting issue here.

Does this function the same on Windows as it does with other operating systems?